examples/README.md

```diff
@@ -1,6 +1,6 @@
 # openshift-ansible usage examples
 
-The primary use of `openshift-ansible` is to install, configure and upgrade OpenShift clusters.
+The primary use of `openshift-ansible` is to install, configure, and upgrade OpenShift on an existing cluster.
```

Comment:
>on an existing cluster

The primary use is to setup a new cluster, updating existing cluster being second. Reconfiguring existing cluster is not really recommended